lib/internal/process.js

```diff
@@ -65,6 +65,13 @@ function setupConfig(_source) {
     if (value === 'false') return false;
     return value;
   });
+  if ( process.config.variables.v8_enable_i18n_support && global.hasOwnProperty('Intl') 
```

Comment:
Style error: space after `(`.  Can you put each check on its own line, followed by the `&&`?  You'll also need to start with:
```js
if (process.config &&
    process.config.variables &&
    // ...
```